src/Microsoft.AspNetCore.OData/ResourceSetContext.cs

```diff
@@ -16,19 +17,20 @@ namespace Microsoft.AspNet.OData
     public partial class ResourceSetContext
     {
         private HttpRequest _request;
+        private IUrlHelper _urlHelper;
 
         /// <summary>
         /// Gets or sets the HTTP request that caused this instance to be generated.
         /// </summary>
         /// <remarks>This signature uses types that are AspNetCore-specific.</remarks>
         public HttpRequest Request
         {
-            get { return _request; }
+            get => _request;
```

Comment:
Stylecop/Code Analysis may complain about this syntax in the build someday.

Reply:
AspNetCore will target the vs2017 or later. It's recommendation. But, I can revert it.